azure-pipelines-e2e-tests-template.yml

```diff
@@ -26,6 +26,13 @@ stages:
             demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
 
         steps:
+        - script: 'curl -k -L -o apache-maven-3.5.3-bin.zip https://archive.apache.org/dist/maven/maven-3/3.5.3/binaries/apache-maven-3.5.3-bin.zip'
```

Comment:
Is maven not available on the linux host ?

Reply:
No looks like 1ES pools do not have maven pre-installed, without this step the pipeline used to fail with the error - https://dev.azure.com/dnceng/public/_build/results?buildId=1573058&view=logs&j=6275ae02-8f8b-5df6-73dc-1bd6a4ec92dd&t=f20d977a-f445-50f9-d91a-fd4b51a6f285

Reply:
Can you install using `apt install` for linux agents ?

Reply:
Done.